src/pages/index.astro

```diff
@@ -399,4 +400,22 @@ const metadata = {
       waste more time! :P
     </Fragment>
   </CallToAction>
+
+  <Donation
+    donationText="هر مبلغی، هر چند کوچک، نشانه‌ای از اعتماد شماست و به ما انگیزه می‌دهد تا با قدرت بیشتری به مسیرمان ادامه دهیم. با حمایت شما، می‌توانیم جامعه فلاتر فارسی را به یکی از بهترین منابع آموزشی در دنیای برنامه‌نویسی تبدیل کنیم."
+    donationButtonText="حمایت مالی"
+    donationLink="/"
+    title={{
+      firstPart: 'با حمایت شما، ',
+      secondPart: 'فلاتر فارسی پرواز می‌کند',
+    }}
+    subtitle="جامعه فلاتر فارسی با عشق و تلاش شما رشد کرده و امروز به نقطه‌ای رسیده که می‌تواند منبعی ارزشمند برای هزاران توسعه‌دهنده باشد. حمایت مالی شما به ما کمک می‌کند تا :"
+    features={[
+      'زیرساخت‌های فنی جامعه را بهبود بخشیم',
+      'دوره‌های آموزشی رایگان بیشتری تولید کنیم',
+      'کیفیت محتوای موجود را ارتقا دهیم',
+      'رویدادهای آموزشی بیشتری برگزار کنیم',
+    ]}
+    imageSrc={donationBgImage}
```

Comment:
I think we don't need to send a parameter to our Donation Component 

Because these parameters are not reusable 

We only use Donation Component in our main page, and it will not change in any other pages or cases 


So let's Remove the parameters and add them to translation files

Reply:
Fixed in new commit